### PR TITLE
Cleanup identity-providers feature (Pt.04)

### PR DESCRIPTION
**Pull request by JayaShakthi97**
### Purpose

- Update some imports from `admin.identity-providers` to `admin.connections`
- Remove unused extension configs
- Move redundant extension configs to utils  

### Related Issues
- https://github.com/wso2/product-is/issues/20580

### Related PRs
- N/A

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: features/admin.connections.v1/components/edit/connection-edit.tsx [hunk 1]
```diff
@@ -237,6 +231,7 @@ export const EditConnection: FunctionComponent<EditConnectionPropsInterface> = (
                 data-testid={ `${ testId }-general-settings` }
                 isReadOnly = { isReadOnly }
                 loader={ Loader }
+                hideIdPLogoEditField
```
Comment (pavinduLakshan) [inline features/admin.connections.v1/components/edit/connection-edit.tsx:234]: Could we know the intention behind removing the config for showing/hiding the idp logo edit field?

Comment (JayaShakthi97) [reply features/admin.connections.v1/components/edit/connection-edit.tsx:234]: The extension config value is redundant and set to true always.

Comment (pavinduLakshan) [reply features/admin.connections.v1/components/edit/connection-edit.tsx:234]: We are planning to bring back the idp logo edit capability[1]. Let's keep this config until that is completed. Having the config will be helpful for us to show/hide the field as required, after the feature is onboarded

[1] https://github.com/wso2/identity-apps/pull/6440

Comment (JayaShakthi97) [reply features/admin.connections.v1/components/edit/connection-edit.tsx:234]: Ack. Please do the necessary cleanup via that PR.

Comment (JayaShakthi97) [reply features/admin.connections.v1/components/edit/connection-edit.tsx]: Added back via b2bf84739c245114ce3b54860c004b050bd98ffe

**Review by savindi7 — APPROVED** [review]

**Pull request merged**